tuneme/static/css/styles.css

```diff
@@ -183,7 +183,7 @@ label {font:normal 15px 'Quicksand', Arial, Helvetica, sans-serif;}
         .magenta .article-copy blockquote, .comment.magenta .by strong, .comment.magenta .report {color:#e60382;}
 
 
-.pagination {padding:15px; text-align: left}
+.pagination {padding:15px; text-align: center;}
 
```

Comment:
@TrevorOctober this will centre the whole pagination, you just want to centre `<---  page 2 of 3 --->`